basemaps/localization.md

```diff
@@ -12,31 +12,81 @@ Protomaps has several localization options for names used in text labels.
 
 <MaplibreMap/>
 
-## Default `name` value
+## Local Names
 
 Protomaps follows OpenStreetMaps's convention where a features's primary name value is is the most common name in the local language(s).
 
 In practice, this is most often a single name value like:
 
 - `London` the locality is represented as a simple key, value pair: `name` = `London`
 
-However, many places have more than one common local languages and Protomaps passes thru OpenStreetMap's convention of concatenating multiple names with a `/` deliminator into a single name value, like:
+However, many places have more than one common local languages and Protomaps passes thru OpenStreetMap's convention of concatenating multiple names with a `/` or `-` deliminator into a single name value, like:
 
 - `Switzerland` the country is represented as a complex key, value pair: `name` = `Schweiz/Suisse/Svizzera/Svizra`
 
-For transnational places involving many countries and languages, like `sea` features, the default name value can get quite long and unweidly!
+For transnational places involving many countries and languages, like `sea` features, the default name value can get quite long and unwiedly!
 
-However, we recommended preferring localized names (see blow) for map labels, and fallback to the default name only when a localized name isn't available.
+### `name`, `name2`, and `name3`
 
-## Localized `name:*` values
+A script or writing system is the way how languages are written. For example, English uses the Latin script, Greek uses the Greek script, and Chinese uses the Han script.
 
-Protomaps structures localized names using the same `name:{language_code}` formatting as OpenStreetMap.
+If a name from OpenStreetMap contains text in more than one script, then Protomaps breaks up the name into segments. There can be up to 3 segments: `name`, `name2`, and `name3`. Each segment should have a unique script. 
```

Comment:
maybe clarify this is the `name` tag "de-facto primary local name" from OSM

Reply:
How are the `name2` and `name3` synthetic properties used in the style?

Is there a downside to overwriting the value of `name` with (implied) `name1`?

Reply:
Does the name splitting only happen when an allow listed delim (`/` or `-`) is observed in the upstream OSM `name` tag?